src/renderer/App.js

```diff
@@ -31,95 +62,56 @@ const App = (props) => {
       }
 
       ipcRenderer.on('IPC_LANGUAGE_CHANGED', handleLanguageChanged)
+      // FIXME: is clean-up necessary in one-shot effects?
```

Comment:
Good question. A simple "debug" ```ipcRenderer.listenerCount('IPC_SHOW_PROJECT_MANAGEMENT')``` always returns ```1```. Not cleaning up the event listener does not seem to be an issue but I think it's good practice to take care about it.